### ISSUE TITLE
Crossroads / Integrated Dynamics

### ISSUE THREAD

**Issue by Linc-42** [EmosewaPixel/The-Pixel-Community-Pack#9]
[Crossroads](https://minecraft.curseforge.com/projects/crossroads-mc?gameCategorySlug=mc-mods&projectID=250231) is a mod i'd like to see integrated into other modpacks. It reminds me of a few of Riekas mods which I sorely miss in the 1.12 age.
While I'm at it I'll throw out [Integrated Dynamics](https://minecraft.curseforge.com/projects/integrated-dynamics?gameCategorySlug=mc-mods&projectID=236307) as well. I personally love this mod, it could be considered a bit OP.
Basically I really enjoy the puzzle like qualities of these mods.

**Comment by KingIonTrueLove** [comment]
Is integrated dynamics really op? I mean without integrated tunnels all you can really do is output redstone signals as far as I'm aware... I would really like to know what cool things you could do with it otherwise

**Comment by EmosewaPixel** [comment]
Hmmm, yeah, Crossroads really doesn't fit in well with other mods. It does it's own thing, like Future Pack.
